akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowThrottleSpec.scala

```diff
@@ -105,12 +105,12 @@ class FlowThrottleSpec extends AkkaSpec {
     }
 
     "send elements downstream as soon as time comes" in Utils.assertAllStagesStopped {
-      val probe = Source(1 to 10).throttle(2, 500.millis, 0, Shaping).runWith(TestSink.probe[Int])
+      val probe = Source(1 to 10).throttle(2, 750.millis, 0, Shaping).runWith(TestSink.probe[Int])
```

Comment:
The idea was to increase all the timings by 0.5 to give more room for variations on the CI server, but maybe I thought about it the wrong way. (The other timings depend on this, so cannot be changed independently).

Couldn't think of a better way to stabilize this test as it is testing timing things.